docs/src/main/paradox/routing-dsl/play-comparison.md

```diff
@@ -0,0 +1,162 @@
+# Compared with Play routes
+
+If you have been using @scala[[Play's routes file syntax](https://www.playframework.com/documentation/2.8.x/ScalaRouting#The-routes-file-syntax)]@java[[Play's routes file syntax](https://www.playframework.com/documentation/2.8.x/JavaRouting#The-routes-file-syntax)] earlier, this page may help you to use the Akka HTTP routing DSL.
+
```

Comment:
I think we need an overview section about the more general differences between play and akka-http routes.

 * internal vs. external DSL
 * what is covered (and what not) by play route files vs. what is covered by Akka HTTP routing

Reply:
I added a more general introduction, but I don't want to go too deep as I think the main purpose of this is the side-by-side illustration.